redis/src/main/scala/zio/redis/api/Scripting.scala

```diff
@@ -0,0 +1,136 @@
+package zio.redis.api
+
+import zio.redis.Input._
+import zio.redis.Output._
+import zio.redis._
+import zio.{ Chunk, ZIO }
+
+trait Scripting {
+  import Scripting._
+
+  /**
+   * Evaluates a Lua script
+   *
+   * @param script Lua script
+   * @param keys keys available through KEYS param in the script
+   * @param args values available through ARGV param in the script
+   * @return redis protocol value that is converted from the Lua type.
+   *         You have to write decoder that would convert
+   *         redis protocol value to a suitable type for your app
+   *
+   * @since 2.6.0
```

Comment:
I'd remove `@since` as it refers to Redis version, which doesn't really matches the lib version.